src/main/webapp/data/userMapData.json

```diff
@@ -315,6 +325,7 @@
     ],
     "Denmark": [
       "Aarhus University",
+      "Aalborg University",
```

Comment:
Opps one more here, let's put it in front.

Also, can you double check by yourself to ensure there's no more disorders in this file.

Reply:
ok, yes I will check .

Reply:
actually I am checking to ensure no more disorder, so should I only change the disorder of names in the list or can change order of names which already in the file?
Please tell me.

Reply:
Everything should be fine for now, since it is not really in the scope of this PR :)